README.md

```diff
@@ -87,286 +93,299 @@ helm uninstall my-release
-| `exporter.nodeSelector` |  Node labels for pod assignment | `{}` |
-| `exporter.tolerations` | Tolerations for pod assignment | `[]` |
-| `exporter.affinity` | Node/Pod affinities | `{}` |
-| `exporter.cacheDuration` | the cache duration for infomation that exporter collected from Harbor | `30` |
-| `exporter.cacheCleanInterval` | cache clean interval for infomation that exporter collected from Harbor | `14400` |
-| `exporter.priorityClassName`                                           | The priority class to run the pod as | |
-| **Metrics** |
-| `metrics.enabled`| if enable harbor metrics | `false` |
-| `metrics.core.path`| the url path for core metrics | `/metrics` |
-| `metrics.core.port` | the port for core metrics | `8001` |
-| `metrics.registry.path` | the url path for registry metrics | `/metrics` |
-| `metrics.registry.port` | the port for registry metrics | `8001` |
-| `metrics.exporter.path` | the url path for exporter metrics | `/metrics` |
-| `metrics.exporter.port` | the port for exporter metrics | `8001` |
-| `metrics.serviceMonitor.enabled`           | create prometheus serviceMonitor. Requires prometheus CRD's | `false` |
-| `metrics.serviceMonitor.additionalLabels`  | additional labels to upsert to the manifest                 | `""`    |
-| `metrics.serviceMonitor.interval`          | scrape period for harbor metrics                            | `""`    |
-| `metrics.serviceMonitor.metricRelabelings` | metrics relabel to add/mod/del before ingestion             | `[]`    |
-| `metrics.serviceMonitor.relabelings`       | relabels to add/mod/del to sample before scrape             | `[]`    |
+| Parameter                                                                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Default                                                                             |     |       |
+| --------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- | --- | ----- |
```

Comment:
Seems there are two redundant columns?

Reply:
Yep, not sure what the `dev` and `1` are there for, seems to be some weird formatting from the past?

![image](https://user-images.githubusercontent.com/11833599/120172551-d93cc780-c20b-11eb-89e6-93484c10a4e7.png)

columns do not match on master either, should I just remove the extra `1` and `dev`

Reply:
Seems the `dev` blocks are from imagetag lines. fixing that :+1:

Reply:
the rouge `1`s were amount of replicas, fix pushed